00:00:47 - ERR - post - (TypeError('can only concatenate str (not "TypeError") to str'),)
00:01:16 - ERR - post - (TypeError('can only concatenate str (not "TypeError") to str'),)
00:03:15 - ERR - post - (TypeError('can only concatenate str (not "TypeError") to str'),)
00:11:53 - ERR - Login - ('Error al obtener los datos del registro - Login',)
00:13:26 - ERR - Login - ('Error al obtener los datos del registro - Login',)
00:14:18 - ERR - Login - ('Error al obtener los datos del registro - Login',)
13:23:05 - ERR - Login - ('Error al obtener los datos del registro - Login',)
13:36:05 - ERR - Login - (AttributeError("'RealDictRow' object has no attribute 'id'"),)
13:38:35 - ERR - Login - (AttributeError("'RealDictRow' object has no attribute 'fk_tb_estudiante'"),)
20:27:37 - ERR - validar - ('No se encontro un usuario registrado en la Senescy - Validar',)
20:27:37 - ERR - signup - ('Error al registrar un nuevo estudiante: ', TypeError("'bool' object is not subscriptable"))
20:28:30 - ERR - signup - ('Error al registrar un nuevo estudiante: ', TypeError('RegistrarDatosUniversitarios.newUniversitario() takes 5 positional arguments but 6 were given'))
21:28:08 - ERR - getRecords - (TypeError('not all arguments converted during string formatting'),)
21:28:08 - ERR - recuperarCuenta - ('No se encontro una cuenta asociada al correo',)
21:29:05 - ERR - getRecords - (UndefinedTable('missing FROM-clause entry for table "jose"\nLINE 1: ...rsitarios_estudiante WHERE correo_institucional = jose.ronqu...\n                                                             ^\n'),)
21:29:05 - ERR - recuperarCuenta - ('No se encontro una cuenta asociada al correo',)
21:35:21 - ERR - getRecords - (UndefinedTable('missing FROM-clause entry for table "jose"\nLINE 1: ...rsitarios_estudiante WHERE correo_institucional = jose.ronqu...\n                                                             ^\n'),)
21:35:21 - ERR - recuperarCuenta - ('No se encontro una cuenta asociada al correo',)
21:38:01 - ERR - getRecords - (UndefinedTable('missing FROM-clause entry for table "test1"\nLINE 1: ...rsitarios_estudiante WHERE correo_institucional = test1.test...\n                                                             ^\n'),)
21:38:01 - ERR - recuperarCuenta - ('No se encontro una cuenta asociada al correo',)
21:38:52 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:41:05 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:41:09 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:42:54 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:44:51 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:48:21 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:48:46 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', KeyError(0))
21:49:22 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', TypeError("'int' object is not subscriptable"))
21:50:17 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', TypeError("string indices must be integers, not 'str'"))
21:52:52 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', AttributeError("'int' object has no attribute 'ToString'"))
21:53:56 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', TypeError("string indices must be integers, not 'str'"))
21:55:16 - ERR - recuperarCuenta - ('Se ha generado un error al crear el token de acceso: ', TypeError("string indices must be integers, not 'str'"))
22:25:19 - ERR - post - (UnboundLocalError("cannot access local variable 'token' where it is not associated with a value"),)
22:26:52 - ERR - post - (UnboundLocalError("cannot access local variable 'token' where it is not associated with a value"),)
22:28:08 - ERR - post - ("Error al Validar el Request -> {'_schema': ['Invalid input type.']}",)
22:38:01 - ERR - post - (ValidationError({'_schema': ['Invalid input type.']}),)
22:38:01 - ERR - post - (ValidationError({'_schema': ['Invalid input type.']}),)
22:41:33 - ERR - activarRecuperacion - ('Se ha generado un error al validar el token: ', DecodeError('It is required that you pass in a value for the "algorithms" argument when calling decode().'))
22:42:11 - ERR - decodeTokenForgotPass - ('Error al tratar de desencriptar el token de acceso',)
22:42:11 - ERR - activarRecuperacion - ('Se ha generado un error al obtener el token',)
22:42:12 - ERR - decodeTokenForgotPass - ('Error al tratar de desencriptar el token de acceso',)
22:42:12 - ERR - activarRecuperacion - ('Se ha generado un error al obtener el token',)
22:42:15 - ERR - decodeTokenForgotPass - ('Error al tratar de desencriptar el token de acceso',)
22:42:15 - ERR - activarRecuperacion - ('Se ha generado un error al obtener el token',)
22:45:18 - ERR - decodeTokenForgotPass - ('Error: El token ha expirado.',)
22:45:18 - ERR - activarRecuperacion - ('Se ha generado un error al obtener el token',)
22:45:19 - ERR - decodeTokenForgotPass - ('Error: El token ha expirado.',)
22:45:19 - ERR - activarRecuperacion - ('Se ha generado un error al obtener el token',)
22:49:16 - ERR - activarRecuperacion - ('Se ha generado un error al validar el token: ', KeyError(0))
23:09:28 - ERR - activarRecuperacion - ('Token decifrado - activarRecuperacion',)
23:09:29 - ERR - activarRecuperacion - ('Datos de la cuenta obtenidos- activarRecuperacion',)
23:10:16 - ERR - activarRecuperacion - ('Token decifrado - activarRecuperacion',)
23:10:16 - ERR - activarRecuperacion - ('Datos de la cuenta obtenidos- activarRecuperacion',)
23:11:47 - ERR - activarRecuperacion - ('Token decifrado - activarRecuperacion',)
23:11:48 - ERR - activarRecuperacion - ('Datos de la cuenta obtenidos- activarRecuperacion',)
23:14:19 - ERR - activarRecuperacion - ('Token decifrado - activarRecuperacion',)
23:14:19 - ERR - activarRecuperacion - ('Datos de la cuenta obtenidos- activarRecuperacion',)
23:14:38 - ERR - activarRecuperacion - ('Token decifrado - activarRecuperacion',)
23:14:39 - ERR - activarRecuperacion - ('Datos de la cuenta obtenidos- activarRecuperacion',)
23:29:22 - ERR - post - (AttributeError("'actualizarCuentaU' object has no attribute 'validate'"),)
23:31:08 - ERR - ExecuteNonQuery - (UndefinedColumn('column "user1" does not exist\nLINE 1: ...tb_datos_universitarios_estudiante SET contrasena=user1 WHER...\n                                                             ^\n'),)
23:31:36 - ERR - post - ('Se ha generado un errro al actualizar la contraseña',)
23:31:50 - ERR - post - ('Se ha generado un errro al actualizar la contraseña',)
23:32:06 - ERR - ExecuteNonQuery - (UndefinedColumn('column "user1" does not exist\nLINE 1: ...tb_datos_universitarios_estudiante SET contrasena=user1 WHER...\n                                                             ^\n'),)
23:32:50 - ERR - post - ('Se ha generado un errro al actualizar la contraseña',)
23:33:02 - ERR - post - ('Se ha generado un errro al actualizar la contraseña',)
